.github/workflows/main.yaml

```diff
@@ -134,6 +134,24 @@ jobs:
             exit 1
           fi
 
+          output=$(helm template oci://ttl.sh/automated-${{ github.run_id }}/replicated --version 0.0.0)
+
+          if ! echo $output | grep -q 'statusInformers: null'; then
+            printf "default statusInformers should be null:\n\n%s\n\n" "$output"
+            exit 1
+          fi
+
+          cat << EOF > test-values.yaml
+          statusInformers: []
+          EOF
+
+          output=$(helm template oci://ttl.sh/automated-${{ github.run_id }}/replicated --version 0.0.0 --values test-values.yaml)
```

Comment:
nit pick: any reason not to use `--set replicated.statusInformers=[]`?

Reply:
saw that too. maybe it's because of how we template it

Reply:
yeah, probably. hmm, this might break in a production install if someone decides to override status informers using helm `set`, right?

Reply:
if they try to override it using `--set replicated.statusInformers=[]`

Reply:
the key issue is that `{{- if .Values.statusInformers }}` evaluates to `true` when using `--set replicated.statusInformers=[]`, but evaluates to `false` when passing the same via a user-provided values file. looking like Helm is interpreting `[]` as a string in the former case and as an array in the latter.

Reply:
Why not use `if not (kindIs "invalid" ...` like we do with the `integration.enabled` field?

Reply:
The result is the same in that case as it's interpreted as a string when `--set replicated.statusInformers=[]` is passed to the cli:
```
$ helm template replicated chart/replicated-0.0.0.tgz --set statusInformers=[] | grep -i statusInformers -A1
    statusInformers:
      '[]'
```